dir.props

```diff
@@ -1,18 +1,23 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Build Tools Version -->
+  <PropertyGroup>
+    <BuildToolsVersion>1.0.21-prerelease</BuildToolsVersion>
+  </PropertyGroup>
```

Comment:
Just for clarity it may be worth adding a comment here that we need a version already on the nuget feed to build the buildtools repo and it cannot be the currently built one.